src/recommendations/events.md

```diff
@@ -7,16 +7,16 @@ When you deploy the `product-recommendations` module, the module tracks the foll
 
 Event |Description
 --- | ---
-`add-to-cart` | Triggered when the shopper adds an item to their cart
-`add-to-wishlist` | Triggered when the shopper adds an item to their wishlist
-`apply-discount-action` | Triggered when the shopper applies a discount to their cart
-`edit-product-qty` | Triggered when the shopper modifies the number of items in their cart
-`initiate-checkout` | Triggered when the shopper clicks the **Proceed to Checkout** button
-`instant-purchase` | Triggered when the shopper selects the **Add to Cart** button on a category page
-`place-order` | Triggered when the shopper clicks the **Place Order** button
-`remove-discount-action` | Triggered when the shopper removes a discount from their cart
-`remove-from-cart` | Triggered when the shopper removes an item from their cart
-`remove-from-wishlist` | Triggered when the shopper removes an item from their wishlist
-`sign-in` | Triggered when the shopper signs into their account
-`sign-out` | Triggered when the shopper signs out of their account
-`view` | Triggered when a shopper views a product details page (PDP)
+`add-to-cart` | A product is added to the cart
+`add-to-wishlist` | A product is added to the wishlist
+`apply-discount-action` | A discount code is applied to the cart
+`edit-product-qty` | A quantity of items in a cart is modified
+`initiate-checkout` | The shopper initiates checkout
+`instant-purchase` | The shopper completes an instant purchase.
+`place-order` | The shopper's order is submitted
+`remove-discount-action` | A discount code is removed from a cart
+`remove-from-cart` | A product is removed from a cart
+`remove-from-wishlist` | A product is removed from a wishlist
+`sign-in` | The shopper signs into their account
```

Comment:
I don't know if Recommendations uses a slightly different vocabulary, but in general, someone with an account is a customer. (Guests either don't have an account or haven't signed in yet.)